app/views/spree/admin/orders/_list.html.haml

```diff
@@ -0,0 +1,16 @@
+%table.payment-list
```

Comment:
this is the payment list so the partial would be better named as _payments_list?